third-party/jquery-ui/jquery.ui.touch-punch.js

```diff
@@ -0,0 +1,180 @@
+/*!
```

Comment:
From where does this file comes from?

Reply:
http://touchpunch.furf.com/
Jquery-ui does not support touch events. Without this patch it won't work on mobile devices.

Reply:
Any reason to not using the npm package?
https://www.npmjs.com/package/jquery-ui-touch-punch

Reply:
My reasoning was to put it to the same place as jquery-ui, since they belong together. We don't use an npm package for jquery-ui either.
The touchpunch code hasn't changed in the last four years, so I don't expect updates.

Reply:
The reason to don't use the `jquery-ui` npm it to have a custom build that's not possible with the npm package...

Reply:
Changed it.